mappings/net/minecraft/client/render/debug/DebugRenderer.mapping

```diff
@@ -2,10 +2,17 @@ CLASS net/minecraft/class_863 net/minecraft/client/render/debug/DebugRenderer
 	CLASS class_864 Renderer
 		METHOD method_20414 clear ()V
 		METHOD method_23109 render (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDJ)V
+			ARG 1 matrix
+			ARG 2 vertices
+			ARG 3 x
+			ARG 5 y
+			ARG 7 z
+			ARG 9 time
 	FIELD field_18777 villageDebugRenderer Lnet/minecraft/class_4207;
 	FIELD field_18778 goalSelectorDebugRenderer Lnet/minecraft/class_4205;
 	FIELD field_19325 raidCenterDebugRenderer Lnet/minecraft/class_4304;
 	FIELD field_20519 gameTestDebugRenderer Lnet/minecraft/class_4503;
+	FIELD field_21547 hiveFarmDebugRenderer Lnet/minecraft/class_4703;
```

Comment:
"Hive farm"...? This debugs bees, their hives, and their flowers. It should just be `beeDebugRenderer`.

Reply:
Missed that in the rename. I'll fix it later.